Dockerfile

```diff
@@ -23,13 +26,21 @@ RUN \
     apt-get install --assume-yes --no-install-recommends 'nodejs=14.*' && \
     apt-get clean && \
     rm --recursive --force /var/lib/apt/lists/*
+
+RUN \
+    . /etc/os-release && \
+    apt-get update && \
+    apt-get install --assume-yes --no-install-recommends
```

Comment:
Here it install nothing but leave package cache in image ?

Reply:
Oups thanks :-)